nautobot/core/settings.py

```diff
@@ -391,6 +391,7 @@
     "nautobot.core",
     "django.contrib.admin",  # Must be after `nautobot.core` for template overrides
     "django_celery_beat",  # Must be after `nautobot.core` for template overrides
+    "django_celery_results",  # Must be after `nautobot.core` for template overrides
```

Comment:
Comment seems misleading - does django-celery-results even have any page templates that we are planning to override?